### PR TITLE
Fix typo in the ConnectorSplitManager section title

### PR DESCRIPTION
**Pull request by francescomucio**
Typo in the title of the ConnectorSplitManger section, it was Manger one `a` was missing.

### CODE REVIEW

File: presto-docs/src/main/sphinx/develop/connectors.rst [hunk 1]
```diff
@@ -38,7 +38,7 @@ to adapt something that is not a relational database (as the Example HTTP
 connector does), you may need to get creative about how you map your
 data source to Presto's schema, table, and column concepts.
 
-ConnectorSplitManger
+ConnectorSplitManager
 ^^^^^^^^^^^^^^^^^^^^
```
Comment (findepi) [inline presto-docs/src/main/sphinx/develop/connectors.rst]: add one `^` to this line too

**Comment by francescomucio** [comment]
I sent it to the cla@prestosql.io, pdf format, signature is on the second page

File: presto-docs/src/main/sphinx/develop/connectors.rst (hunk 1 above)
Comment (francescomucio) [reply presto-docs/src/main/sphinx/develop/connectors.rst]: done

**Comment by electrum** [comment]
@cla-bot check

**Comment by electrum** [comment]
@francescomucio thanks! Can you squash the two commits together, since they are part of the same change?

**Comment by findepi** [comment]
Merged as 03c1e9dfea45f13052fdb5835e412a7bdd573349, thanks!

**Pull request closed without merging**